### DISCUSSION TITLE
Display Time option needs higher possible value

### DISCUSSION THREAD

**Discussion by KodeXall** [jonaskohl/CapsLockIndicator#127] [Ideas]
Right now a max number of milliseconds that can be set at

Notification > Display time

is 1999. I do not understand why user should be limited to just 2 seconds

I personally would benefit from more time, as I often [and even right now] write stuff with my mini handheld keyboard that obscures my view of the screen when I am laying on the bed. That keyboard also happens to have highlighted keys as otherwise it would be useless in dark - and to help me fall asleep the color I have chosen for its keys is red. And so when I combine this with red-themed Notification for when my CapsLock is ON, I tend to not notice the state change in such relatively short period of time as 2 seconds- because only the more keys I press does my keyboard slightly moves in the line of sight between my head and the monitor [thus allowing me to see Notification

Thus I propose to give the user if not unlimited time then at least up to 5 or 10 seconds

**Comment by jonaskohl** [comment]
This would bring difficulties with the trackbar, as it has to have a reasonably small maximum value to allow for some level of granularity. However, I could clamp the trackbar's value and allow the spinner to go to a larger maximum (e.g. 65536 or something similar)
